### PR TITLE
Remove unnecessary dependency & decompile the entire APK

### LINKED ISSUE #30: ApkLeaks does not decompile all DEX files
The code reveals that ApkLeaks only decompiles `classes.dex`, omitting any other DEX files: https://github.com/dwisiswant0/apkleaks/blob/master/apkleaks/apkleaks.py#L92

Since `jadx` is able to decompile an entire APK file, I suggest to pass the following arguments to `jadx`:
```python
args = [self.jadx, self.file, "-d", self.tempdir, "--deobf"]
```

### PR DESCRIPTION
**Pull request by U039b**
- I have removed `numpy` as its use was limited to remove duplicates in findings list, using `list(set(findings))` does the work well.
- I have changed the arguments passed to `jadx` in order to decompile the entire APK. That way, the extraction of the leaks occurs on the entire content of the APK and obviously finds more things :]. (close #30)

### CODE REVIEW

File: apkleaks/apkleaks.py [hunk 1]
```diff
@@ -85,35 +84,25 @@ def integrity(self):
...
-		return (numpy.unique(x))
-
 	def finder(self, pattern, path):
 		matcher = re.compile(pattern)
 		found = []
 		for path, _, files in os.walk(path):
 			for fn in files:
 				filepath = os.path.join(path, fn)
-				if mimetypes.guess_type(filepath)[0] is None:
-					continue
```
Comment (dwisiswant0) [inline apkleaks/apkleaks.py:110]: May I know why this part was removed?

Comment (dwisiswant0) [reply apkleaks/apkleaks.py:110]: Oh, being catched on L98-105. Got it.

**Review by dwisiswant0 — APPROVED** [review]
Hi, @U039b. This is neat! Didn't notice those `numpy` part actually. Thanks for your contributions! Merging now.

**Comment by U039b** [comment]
You are welcome :) I think I will do another PR creating a `setup.py` file to make your project easier to install with `pip`.

**Pull request merged**